app/utils/HttpResponseHelper.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import uk.gov.hmrc.http._
+import play.api.http.Status._
+
+trait HttpResponseHelper extends HttpErrorFunctions {
+
+  implicit val httpResponseReads: HttpReads[HttpResponse] = (method: String, url: String, response: HttpResponse) => response
+
+  def handleErrorResponse(httpMethod: String, url: String)(response: HttpResponse): Nothing =
```

Comment:
Return type is "Nothing"?

Reply:
Once we created the error page, we will redirect to that page... this also temporary.